src/backend/api/auth.ts

```diff
@@ -12,3 +12,15 @@ export const logOut = async () => {
 
 export const signInWithProvider = (provider: string) =>
   ipcRenderer.send('openExternalUrl', `${DEV_PORTAL_URL}/oauth/${provider}`)
+
+export const authConnected = () => ipcRenderer.send('authConnected')
+export const authDisconnected = () => ipcRenderer.send('authDisconnected')
```

Comment:
What do you think about renaming these to `notifyWindowsAuthConnected` and `notifyWindowsauthDisconnected`?

This is what we're doing right?

Reply:
it is notifying the backend only. the backend decides whether to notify other windows or not